src/commands/health-check.ts

```diff
@@ -0,0 +1,37 @@
+import {flags} from '@oclif/command'
+import * as colors from 'colors'
+import Constants from '../services/constants'
+import healthCheck from '../utils/health-checker'
+import PercyCommand from './percy-command'
+
+export default class HealthCheck extends PercyCommand {
+  static description = 'Determins if the Percy Agent process is currently running'
+  static hidden = true
+
+  static flags = {
+    port: flags.integer({
+      char: 'p',
+      default: Constants.PORT,
+      description: 'port',
+    }),
+  }
+
+  static examples = [
+    '$ percy healthcheck',
+    '$ percy healthcheck --port 6884',
+  ]
+
+  async run() {
+    await super.run()
+
+    // If Percy is disabled or is missing a token, gracefully exit here
+    if (!this.percyWillRun()) { this.exit(0) }
```

Comment:
not super sure if this is actually relevant in this particular context. It's useful for gracefully disabling a command if Percy isn't going to run, but this is sort of different. `this.percyWillRun()` will also check for a `PERCY_TOKEN` but that's not relevant here.

Reply:
Yeah I debated if I should add it. It'd probably fail no matter what -- so I can remove it.